core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -333,6 +333,49 @@ UpdateCenter.PluginCategory.view=Views
 UpdateCenter.PluginCategory.must-be-labeled=Uncategorized
 UpdateCenter.PluginCategory.unrecognized=Misc ({0})
 
+UpdateCenter.compatWarning=\
```

Comment:
these are all duplicated with `table.properties` so that I can access them in java. 😢